src/openrct2/GameState.h

```diff
@@ -39,6 +39,7 @@ namespace OpenRCT2
         money64 ParkValue;
         money64 ParkValueHistory[FINANCE_GRAPH_SIZE];
         uint8_t ParkRatingHistory[32];
+        money64 CompanyValue;
```

Comment:
i would try keep the order in the structure the same as the order in parkfile.cpp file. I.e. move company value to just after park value.